utilities/wannier2sparse/include/operator_utils.hpp

```diff
@@ -128,6 +141,18 @@ namespace oputil
 		 return op[1];
 	 };
 
+         /**
+	 * Extract the direction of the projection fron the operator label. This function does not check if the operator belong to a projection operator. 
+	 *
+	 * @param[out] direction  The direction of the projection which can be = +1,-1;
+	 * @param[in]  op string containing the operator label.
+	 */
+	 inline
+	 char projection_direction(string op)
+	 {
+		 return op[1];
+	 };
+

```

Comment:
This whole function could be scraped in favour of using the velocity_direction function which does the same,

Reply:
Let's keep it for the moment. Once the code is finished, we can developed some testing units and then can safely change things.